src/webviews/apps/home/components/home-app.ts

```diff
@@ -0,0 +1,105 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { repeat } from 'lit/directives/repeat.js';
+import { when } from 'lit/directives/when.js';
+import type { OnboardingState } from '../../../home/protocol';
+import type { OnboardingItemConfiguration } from '../model/gitlens-onboarding';
+import { onboardingConfiguration } from '../model/gitlens-onboarding';
+
+@customElement('gl-home-app')
+export class GlHomeApp extends LitElement {
+	@property({ type: Object })
+	state?: OnboardingState;
+
+	static override readonly styles = css`
+		gl-progress-tracker {
+			padding: 0 20px;
```

Comment:
I would remove this padding and let the actual home page (home.scss) control the spacing around the component for better flexibility.

Reply:
done

Reply:
I had to revert it and move the padding to the accordion to make the hovered(focused) style full-width